src/main/java/cafeloader/Cafe_ElfExtension.java

```diff
@@ -311,11 +311,10 @@ private void processRplFileInfo(ElfLoadHelper elfLoadHelper, ElfSectionHeader se
 				Address tagAddress = fileInfoAddr.add(tagOffset);
 				while (true) {
 					Data d = elfLoadHelper.createData(tagAddress, TerminatedStringDataType.dataType);
-					int length = d.getLength();
-					if (length == 0) {
+					if (d == null || d.getLength() <= 1) { // empty string has a length of 1 (just a null terminator)
```

Comment:
I think a better fix would be to check `"if d is null or d.getLength() == 0 or d[0] == 0`. (I don't know the syntax for it in Java.)